parsers/ENTSOE.py

```diff
@@ -407,7 +407,13 @@ def query_ENTSOE(session, params, target_datetime=None, span=(-48, 24)):
     params['periodEnd'] = target_datetime.shift(hours=span[1]).format('YYYYMMDDHH00')
     if 'ENTSOE_TOKEN' not in os.environ:
         raise Exception('No ENTSOE_TOKEN found! Please add it into secrets.env!')
-    params['securityToken'] = os.environ['ENTSOE_TOKEN']
+    tokens = [os.environ['ENTSOE_TOKEN']]
```

Comment:
Suggestion:
```suggestion
    tokens = os.environ['ENTSOE_TOKEN'].split(',')
```

That way we can pass one or multiple ENTSOE_TOKEN separated by a comma